### PR TITLE
UI design for Math-Magicians project pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
- Designed the user interface for the math-magicians project
- create child component as required

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div className="fourth-row">
+        <span>1</span>
+        <span>2</span>
+        <span>3</span>
+        <span className="special">+</span>
+      </div>
+    </div>
+    <FirthRow />
+  </div>
+);
```
Comment (DeliceLydia) [inline src/components/Calculator.js:42]: - [ ] For this project the `Calculator.js` component should be a functional component

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @AleWaweru, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The app is running smoothly
-  Git workflow followed
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (AleWaweru) [inline src/components/Calculator.js]: done.

**Review by ishimwezachee — APPROVED** [review]
Hi @AleWaweru , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image
## Highlights
✅All linters are checked.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**